src/styles/global.scss

```diff
@@ -14,6 +14,86 @@
   }
 }
 
+html.is-extension {
+  width: $extension-width;
+  height: $extension-height;
+
+  // fixes issue with extension not rendering properly
+  ion-tabs,
+  .tabs-inner,
+  ion-router-outlet,
+  .ion-page {
+    contain: layout style !important;
+  }
+
+  body {
+    position: unset;
+  }
+}
+
+// When scanner is active we need to hide everything else
+// because the scanner will show-up behind the webview
+body.scanner-active {
+  background-color: transparent;
+
+  --background: transparent;
+  --ion-background-color: transparent;
+  --screen-bg-color: transparent;
+
+  .app-wrapper {
+    --screen-bg-color: transparent;
+
+    background: transparent;
+  }
+
+  .ionic-wrapper {
+    background-color: transparent;
+  }
+
+  #app {
+    background: transparent;
+    position: absolute;
+    z-index: 2;
+    width: 100%;
+  }
+
+  ion-content,
+  .ion-padding {
```

Comment:
Name `ion-padding` suggest it has something to do with paddings. But below you are adding some stuff related to backgrounds. Looks really random.